galaxy_ng/app/settings.py

```diff
@@ -137,6 +137,8 @@
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
 LOCALE_PATHS = (os.path.join(BASE_DIR, 'locale'),)
 
+GALAXY_ENABLE_VIEW_ONLY_ACCESS = False
```

Comment:
I'm trying to think about the best way to set up the configuration for this. I'm leaning towards renaming this to `GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS`, since this doesn't enable access to containers. I know this wasn't a requirement from product but I would also like to add a `GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_DOWNLOAD` option that lets people download collections with an anonymous user as well. Don't know if customers want this or not, but doesn't hurt to give people options.

This opens the door for enabling view only EE access as well if we want.